tests/Unit/Service/OsGraphRecallTest.php

```
@@ -0,0 +1,356 @@
+            {
+                return new Edge('e-new', $fromId, $toId, $relation);
+            }
+
+            public function updateNode(string $id, ?string $title = null, array $properties = []): ?Node
+            {
+                return $this->node($id);
+            }
+
+            public function subgraph(string $nodeId, int $depth = 1): array
+            {
+                return $this->neighborhood($nodeId);
```

PHPStan (2.2.8)

[error] Method Semitexa\Weave\Domain\Contract\GraphStoreInterface@anonymous/tests/Unit/Service/OsGraphRecallTest.php:260::subgraph() should return array{nodes: list<Semitexa\Weave\Domain\Model\Node>, edges: list<Semitexa\Weave\Domain\Model\Edge>} but returns array{node: Semitexa\Weave\Domain\Model\Node|null, edges: list<Semitexa\Weave\Domain\Model\Edge>, neighbors: list<Semitexa\Weave\Domain\Model\Node>}.
Array does not have offset 'nodes'.

(return.type)